plugins/meta/vrf/vrf.go

```diff
@@ -130,6 +139,21 @@ CONTINUE:
 		}
 	}
 
+	// Apply all saved routes for the interface that was moved to the VRF
+	for _, route := range routes {
+		r := route
+		// Modify original table to vrf one,
+		// equivalent of 'ip route add <address> table <int>'.
+		r.Table = int(vrf.Table)
+		err = netlink.RouteAdd(&r)
+		if err != nil {
+			// If route is already present, returned error is "file exists"
+			if !strings.Contains(fmt.Sprintf("%v", err), "file exists") {
+				return fmt.Errorf("error while adding route \"%s\": %v\n", r, err)
```

Comment:
no need to add \n and perhaps something like "failed adding ..." or "could not add ..." would be more consistent with other errors in this file